app/src/main/java/org/oppia/android/app/home/recentlyplayed/RecentlyPlayedFragmentPresenter.kt

```diff
@@ -40,8 +40,6 @@ class RecentlyPlayedFragmentPresenter @Inject constructor(
   private val explorationCheckpointController: ExplorationCheckpointController,
   @StoryHtmlParserEntityType private val entityType: String
 ) {
-  // TODO(#3479): Enable checkpointing once mechanism to resume exploration with checkpoints is
-  //  implemented.
```

Comment:
Nothing is actually being changed here--how is the feature being enabled?

Reply:
checkpointing was enabled in PR (#3672), I have linked that PR with this issue.

Reply:
Ah, we actually have a TODO check in place now that will reopen issues if they're closed with TODOs still checked into the codebase. I actually suggest that you remove the TODOs in the same PR that's resolving them since that's what's normally expected.

Reply:
Looks resolved now.